add_me_to_your_PATH/Dockerfile.template

```diff
@@ -116,27 +149,23 @@ RUN --mount=type=secret,id=github_token \
 
 FROM base as project
 
+ENV JULIA_PROJECT /JuliaProject
+
 # copy over artifacts generated during the `precompile-image` stage
-COPY --from=precompile-image /JuliaProject/ /JuliaProject/
+COPY --from=precompile-image /JuliaProject/ ${JULIA_PROJECT}/
```

Comment:
shoudl we use `${JULIA_PROJECT}` here too?  looks like it's used in the precompile-image stage

Reply:
We could. The reason I didn't is that the two directories could use different names. In this case they just happen to be the same